packages/ketcher-react/src/script/ui/views/components/ContextMenu/menuItems/BondMenuItems.tsx

```diff
@@ -29,7 +29,7 @@ const BondMenuItems: FC<MenuItemsProps> = (props) => {
           : 'Edit...'}
       </Item>
 
-      {nonQueryBondNames.map((name) => {
+      {nonQueryBondNames.slice(1).map((name) => {
```

Comment:
My proposal is to add additional variable for this. I guess, it will be unclear for others, why first item is omitted